lightning/src/ln/channelmanager.rs

```diff
@@ -10364,7 +10364,7 @@ where
 				};
 
 				match response {
-					Ok(invoice) => responder.respond(OffersMessage::Invoice(invoice)),
+					Ok(invoice) => responder.respond_with_reply_path(OffersMessage::Invoice(invoice)),
```

Comment:
Note that once you rebase, you'll need to add an `OffersContext` here.

Reply:
Introduced in [pr3163.02](https://github.com/shaavan/rust-lightning/commits/pr3163.02)
Thanks for the pointer!